Source/Scene/LabelCollection.js

```diff
@@ -371,6 +399,17 @@ define([
          * @default false
          */
         this.debugShowBoundingVolume = defaultValue(options.debugShowBoundingVolume, false);
+
+        this._removeEventFunc = undefined;
```

Comment:
Yup. If you look at `Event`, a function to remove a listener from an event is returned when a listener is added:
https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/Event.js#L76